### DISCUSSION TITLE
Tiling Shell v14.1 | Support different layouts accross workspaces and more!

### DISCUSSION THREAD

**Discussion by domferr** [domferr/tilingshell#175] [Announcements]
- **New**: It is now possible to have a different layout for each workspace. With this addition you can now select your favorite layout for each workspace of each monitor. Closes #137 #153 #118 #47 (one of the most requested feature, isn't it? :smile:)

https://github.com/user-attachments/assets/41226602-5950-47d1-bbf6-3d7ff3e265fb

- **New**: You can choose a key to be pressed to deactivate the tiling system while moving a window. Pretty useful if you prefer the tiling system _always on_ (i.e. without activation key) but sometimes you want to move a window without using the tiling system. Closes #164 #141 #130 
- **New**: When you export layouts or settings, the file chooser now comes with the name pre-filled.
- **Fix**: Improved reliability of windows borders. Closes #171

<div align="center">

![](https://img.shields.io/badge/GNOME-42--47-e04196)
[![](https://img.shields.io/badge/Request_a_Feature-8A2BE2)](https://github.com/domferr/tilingshell/issues/new?assignees=&labels=&projects=&template=feature_request.md&title=)
[![](https://img.shields.io/badge/Report_a_Bug-blue)](https://github.com/domferr/tilingshell/issues/new?assignees=&labels=&projects=&template=bug_report.md&title=)
[![kofi](https://img.shields.io/badge/Donate-on_Ko--fi-purple?logo=ko-fi)](https://ko-fi.com/domferr)

</div>

## How to install

:star: This update of Tiling Shell is landing on GSE too!
<div align="center">
  <a href="https://extensions.gnome.org/extension/7065/tiling-shell/" >
      <img src="https://img.shields.io/badge/Install%20from-extensions.gnome.org-4A86CF?style=for-the-badge&logo=Gnome&logoColor=white"/>
  </a>
</div>

## Manual installation

There are two builds and you need only one of them, according to your GNOME Shell version:
- For GNOME Shell versions 45, 46 and 47: `tilingshell@ferrarodomenico.com.zip`
- For GNOME Shell versions from 42 to 44: `GNOME.42-44.tilingshell@ferrarodomenico.com.zip`

To know your GNOME Shell version, run `gnome-shell --version`.

Extract the downloaded archive. Copy its content to `~/.local/share/gnome-shell/extensions/tilingshell@ferrarodomenico.com` directory. You need to reload GNOME Shell afterwards (e.g. by logging out). Then you can enable the extension:
```
/usr/bin/gnome-extensions enable tilingshell@ferrarodomenico.com
```

<hr /><em>This discussion was created from the release <a href='https://github.com/domferr/tilingshell/releases/tag/14.1.0'>Tiling Shell v14.1 | Support different layouts accross workspaces and more!</a>.</em>

**Comment by rocchidavide** [comment]
Thank you Domenico, great feature release this one! :rocket:

**Reply by domferr** [reply]
Thanks a million, I'm so happy you like it!

**Comment by Milli189** [comment]
Hey thank you man you're doing Gods work!

The only thing I can say is that when you import & export layouts you get the old gtk file picker and not the new nautilus one
![image_2024-11-04_133147745](https://github.com/user-attachments/assets/9ac9d1e0-dedc-43d5-ac29-d2ba2781aec4)

edit: I'm on gnome 47
